tests/ui/feature-gates/unstable-attribute-rejects-already-stable-features.rs

```diff
@@ -0,0 +1,11 @@
+//! Ensure #[unstable] doesn't accept already stable features
+//@ check-fail
```

Comment:
Nit: I think you don't need `check-fail` because by default that's what ui tests do.